packages/desktop-client/src/components/mobile/transactions/TransactionList.tsx

```diff
@@ -185,13 +207,21 @@ export function TransactionList({
   );
 }
 
-function SelectedTransactionsFloatingActionBar({ transactions, style }) {
+type SelectedTransactionsFloatingActionBarProps = {
+  transactions: readonly TransactionEntity[];
+  style?: CSSProperties;
+};
+
+function SelectedTransactionsFloatingActionBar({
+  transactions,
+  style = {},
+}: SelectedTransactionsFloatingActionBarProps) {
   const editMenuTriggerRef = useRef(null);
   const [isEditMenuOpen, setIsEditMenuOpen] = useState(false);
   const moreOptionsMenuTriggerRef = useRef(null);
   const [isMoreOptionsMenuOpen, setIsMoreOptionsMenuOpen] = useState(false);
   const getMenuItemStyle = useCallback(
```

Comment:
I think this would also work instead of exporting the type
```suggestion
  const getMenuItemStyle: ComponentProps<typeof Menu>['getItemStyle'] = useCallback(
```

Reply:
@joel-jeremy Thanks for the second review and suggestion! Unfortunately the proposed change doesn't seem to work. 😢 
<img width="1296" alt="image" src="https://github.com/user-attachments/assets/a67ea185-979d-4ad5-bff4-28cb9958ea3c" />

I believe it is because the type of `'NameType'` in the first argument of the Menu's `'getItemStyle'` is a generic that bounds to the actual `Menu` instance (i.e. TS resolving the type from the `items` the `Menu` instance was passed). Whereas it is `unknown` type when it is not bounded to the `Menu` instance. 

I am open to other suggestion, but so far setting this anonymous function with generic type is only way I get typing to work.